src/plugins/condition/utils/evaluator.js

```diff
@@ -1,7 +1,31 @@
-export const computeConditionForAny = (args) => {
-    return false;
+/**
+ * Returns true only if at least one of the results is true
+ **/
+export const computeConditionForAny = (resultMap) => {
+    let result = false;
+    for (let key in resultMap) {
+        if (resultMap.hasOwnProperty(key)) {
+            result = resultMap[key];
+            if (result) {
+                break;
+            }
+        }
+    }
+    return result;
 };
 
-export const computeConditionForAll = (args) => {
-    return false;
+/**
+ * Returns true only if all the results are true
+ **/
+export const computeConditionForAll = (resultMap) => {
+    let result = false;
+    for (let key in resultMap) {
+        if (resultMap.hasOwnProperty(key)) {
+            result = resultMap[key];
+            if (!result) {
+                break;
+            }
+        }
+    }
+    return result;
```

Comment:
The code for `computeConditionForAll()` and `computeConditionForAny()` is identical except for the line that checks the `result`. Instead of repeating the same code, it would be better to define a common function and pass the trigger `any` or `all` depending on which of the 2 functions is calling it.

Reply:
I did this mostly for clarity than anything else as I mentioned above for Joel. I suppose since both of you noticed, I'll merge the 2.

Reply:
Done.